features/features.go

```diff
@@ -309,3 +252,57 @@ func FromContext(ctx context.Context) *Features {
 	}
 	return nil
 }
+
+func (f *Features) Adjust() ([]string, error) {
```

Comment:
I think having "flags" appear in this method name would make it a bit more readable

```suggestion
func (f *Features) processFlags() ([]string, error) {
```